packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-email/main/index.md

```diff
@@ -28,34 +27,24 @@ Understand how to implement a user sign-in flow with password and email factors.
 
 ---
 
-## Configuration updates
+## Overview
 
-This use case requires the password and email factors.
+With the Embedded SDK, your app can verify a user's identity using first a password and then the email authenticator. The email authenticator can complete its verification using [magic links](/docs/guides/email-magic-links-overview/aspnet/main/), a one-time password (OTP), or a combination of both.
 
 <div class="half">
 
-![Displays a diagram of the required password and email factors](/img/oie-embedded-sdk/factor-password-email.png)
+![A representation of the required password and email factors](/img/oie-embedded-sdk/factor-password-email.png)
```

Comment:
The image is sized to small.

Reply:
Are you sure?  Moving up to three-quarters width makes it look very large indeed.

Reply:
Updated

Reply:
I agree the image looks somewhat too large now. Can include a size in the middle of those two sizes? If not then keep as is for now.